docs/_future/deploy_your_app.md

```diff
@@ -7,10 +7,212 @@ layout: tutorial
+
+* App name: `aqueous-escarpment-85734`
+* Web address: `https://aqueous-escarpment-85734.herokuapp.com/`
+* Git remote: `https://git.heroku.com/aqueous-escarpment-85734.git`
+
+#### Add environment variables
+
+The tokens being used for your app (and any other environment variables) can be added to your deployed environment using the following commands:
+
+```sh
+$ heroku config:set SLACK_APP_TOKEN=xapp-1-70k3n-h3r312xdp
+$ heroku config:set SLACK_BOT_TOKEN=xoxb-2b07-4pp5hv3f3ll85
+```
+
+These tokens and variables don't necessarily have to match those used in development. For instance, you may want to use tokens for a different app or a more verbose logging output in production. This is where you would set that.
+
+### Prepare your app for Heroku {#prepare-your-app}
+
+No changes to your app should be needed to deploy, but a few configurations must be made to make a deployment successful.
+
+#### Add a Procfile
+
+To specify the start command of your app for Heroku, [a special file called **Procfile**](https://devcenter.heroku.com/articles/procfile) must be created.
+
+The contents of this file will vary depending if you are using Socket Mode or not, but in both cases it will be named `Procfile` and live in your app's root directory.
+
+If you are using Socket Mode, your `Procfile` should contain the following:
+```sh
+worker: node app.js
+```
+
+A Slack app using default web connections will have the following `Procfile`:
+
+```sh
+web: node app.js
+```
+
+With this new file saved, go ahead and commit it to your Git repository:
+
+```sh
+$ git add Procfile
+$ git commit -m "Add Procfile"
+```
+
+#### Add the Git remote
+
+Code is deployed to Heroku by pushing code from your local repository to the Git remote from when [you created an app](#create-a-heroku-app).
+
+You can check if this remote was automatically added by the Heroku CLI when your app was created like so:
+
+```sh
+$ git remote -v
+# heroku  https://git.heroku.com/aqueous-escarpment-85734.git (fetch)
+# heroku  https://git.heroku.com/aqueous-escarpment-85734.git (push)
+```
+
+But if there is no `heroku` remote, you can add one as follows:
+
+```sh
+$ git remote add heroku https://git.heroku.com/aqueous-escarpment-85734.git
+```
+
+### Deploy your app {#deploy-your-app}
+
+After creating and configuring your Bolt and Heroku app, you're ready for deployment!
+
+> 💾 Any [triggers created with the CLI](https://api.slack.com/future/triggers#create_cli) for the local "(dev)" version of your app will continue to fire after deploying your app!
+
+#### Push your code
+
+But before you can deploy, you should ensure that you have committed all of the source code your app needs to run!
+
+You can see unstaged changes using `git status` and commit any changes using the same commands used for **Procfile**.
+
+After adding and committing the source code of your app to the `main` branch of your repo, push it to your Heroku remote to start a deployment:
+
+```sh
+$ git push -u heroku main
+```
```

Comment:
Could we add some messaging here to indicate what a successful deployment looks like?

Reply:
Definitely something nice to add, good call! I pasted the end of some logs from a past deploy below (and added a few words) to possibly serve as this messaging!